src/widgets/immunizations/immunization-widget-config-schema.ts

```diff
@@ -1,36 +1,38 @@
-import { validators } from "@openmrs/esm-config";
+import { validators, Type } from "@openmrs/esm-config";
 
 export default {
   vaccinesConceptSet: {
-    validators: [validators.isString],
-    description:
+    _type: Type.String,
+    _description:
       "This needs to be a uuid or concept mapping which will have all the possible vaccines as set-members."
   },
   sequenceDefinitions: {
-    description:
+    _type: Type.Array,
+    _description:
       "Doses/Schedules definitions for each vaccine configured if applicable. If not provided the vaccine would be treated as a vaccine without schedules",
-    arrayElements: {
+    _elements: {
       vaccineConceptUuid: {
-        validators: [validators.isUuid],
-        description: "The UUID of the individual vaccine concept"
+        _validators: [validators.isUuid],
```

Comment:
Should this be `_type: Type.UUID` or `_type: Type.ConceptUuid` ?

Reply:
Yes! Good catch, thanks.